mozilla_schema_generator/glean_ping.py

```diff
@@ -248,7 +252,14 @@ def _get_ping_data_and_dependencies_with_default_metadata(self) -> Dict[str, Dic
                 GleanPing.apply_default_metadata(
                     dependency_ping.get("moz_pipeline_metadata"), default_metadata
                 )
+                # ping-level properties take priority over repo defaults
```

Comment:
```suggestion
                # app-level properties take priority over repo defaults
```
typo?

Reply:
They're ping properties set at the app-level which take priority over the repo/app default.  I can change it to `app-level ping properties take priority over the app defaults` but it's still a bit confusing